nima_core/api/server.py

```
@@ -284,11 +284,38 @@ async def _nima_lifespan(app: FastAPI):
 
+    # Hub-side Apache AGE NER pipeline: ensure graph + start polling worker.
+    try:
+        from nima_core.config import NIMA_HUB_NER_ENABLED
+
+        if NIMA_HUB_NER_ENABLED:
+            from nima_core.federation import pg_graph as _pg_graph
+            from nima_core.federation.hub_ner_worker import HubNERWorker
+
+            try:
+                _pg_graph.ensure_graph_exists()
+            except Exception as _graph_exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)